.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -180,6 +180,7 @@ Cfml
 Cfoo
 cfz
 cgi
+chainindentation
```

Comment:
Please remove this, itis not required anymore

Reply:
removed